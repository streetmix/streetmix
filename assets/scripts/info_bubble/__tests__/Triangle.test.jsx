/* eslint-env jest */
import React from 'react'
import { render } from '@testing-library/react'
import Triangle from '../Triangle'

describe('Triangle', () => {
  it('renders an unhighlighted triangle by default', () => {
    const { asFragment } = render(<Triangle />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('renders an highlighted triangle', () => {
    const { asFragment } = render(<Triangle highlight={true} />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('renders an unhighlighted triangle', () => {
    const { asFragment } = render(<Triangle highlight={false} />)
    expect(asFragment()).toMatchSnapshot()
  })
})
