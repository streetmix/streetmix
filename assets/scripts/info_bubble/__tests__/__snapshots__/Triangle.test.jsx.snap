// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Triangle renders an highlighted triangle 1`] = `
<DocumentFragment>
  <div
    class="info-bubble-triangle info-bubble-triangle-highlight"
  />
</DocumentFragment>
`;

exports[`Triangle renders an unhighlighted triangle 1`] = `
<DocumentFragment>
  <div
    class="info-bubble-triangle"
  />
</DocumentFragment>
`;

exports[`Triangle renders an unhighlighted triangle by default 1`] = `
<DocumentFragment>
  <div
    class="info-bubble-triangle"
  />
</DocumentFragment>
`;
